### ISSUE TITLE
False positive: Section title rule triggers inside source code blocks

### ISSUE THREAD

**Issue by raifdmueller** [docToolchain/asciidoc-linter#52]
## Description

The heading rule (section title should be preceded by a blank line) triggers on content inside AsciiDoc source code blocks (`[source,asciidoc]` with `----` delimiters).

## Steps to Reproduce

Given this AsciiDoc file:

```asciidoc
Some text before:

[source,asciidoc]
----
=== S-01: Buchung manuell erfassen

Status:: Offen
----
```

Running `asciidoc-linter file.adoc` reports:

```
✗ file.adoc, line 5: Section title should be preceded by a blank line
```

## Expected Behavior

Content inside `----` delimited blocks should be skipped by all rules, since it is literal/source content, not actual AsciiDoc structure.

## Actual Behavior

The linter parses `=== S-01:` inside the code block as a real section title and flags it.

## Environment

- asciidoc-linter 0.1.0 (installed from git main branch)

**Comment by rdmueller** [comment]
## Analysis

Already fixed on `main` by #56 (commit 2ded7ad). `asciidoc_linter/blocks.py` now detects verbatim blocks (`----`, `....`, `++++`, `////`) centrally. The parser and every rule (`mask_verbatim_blocks`, `find_code_block_content_lines`) skip their content.

Verified with the reproduction from this issue: `asciidoc-linter --format plain` reports `✓ No issues found`, exit code 0.

Suggestion: close as completed. No further PR needed.

---
_Generated by [Claude Code](https://claude.ai/code)_
